src/backends/zora-indexer-v1/ZoraIndexerV1DataSource.ts

```diff
@@ -455,11 +456,14 @@ export class ZoraIndexerV1DataSource implements ZoraIndexerV1Interface {
     return object;
   }
 
-  loadNFT = async (tokenContract: string, tokenId: string) => {
-    return await this.nftGraphDataLoader.load(`${getAddress(tokenContract)}-${tokenId}`);
+  loadNFT = async ({ contract, id }: NFTIdentifier) => {
+    return await this.nftGraphDataLoader.load(`${getAddress(contract)}-${id}`);
   };
-  loadNFTs = async (tokenContractAndId: readonly string[]) => {
-    return await this.nftGraphDataLoader.loadMany(tokenContractAndId);
+
+  loadNFTs = async (nfts: readonly NFTIdentifier[]) => {
+    return await this.nftGraphDataLoader.loadMany(
+      nfts.map((nft) => `${getAddress(nft.contract)}-${nft.id}`)
```

Comment:
note: this is the correct `getAddress()`

Reply:
+